code/__defines/flags.dm

```diff
@@ -33,6 +33,7 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define OBJ_FLAG_ANCHORABLE              0x0001 // This object can be stuck in place with a tool
 #define OBJ_FLAG_CONDUCTIBLE             0x0002 // Conducts electricity. (metal etc.)
 #define OBJ_FLAG_INVINCIBLE				 0x0004	// This object ignores all damage mechanics, and can't be broken, damaged etc
+#define OBJ_FLAG_BEING_ABSORBED			 0x0008	// This object is currently being absorbed as biomass.
```

Comment:
This definitely doesn't need an object flag. this kind of special case is why we have extensions. Just stick a special extension on an object while its being absorbed, and check for the presence of that before absorbing it

Reply:
I felt like a blank extension being used like a flag for this one case is too redundant, a flag like this is super cheap to use and equally easy to check for.

Reply:
It is exactly the opposite. Yes flags are cheap to use, thats why they're reserved for performance critical applications. By adding a new flag there you're reserving it forever and leaving one less potential flag for other, broader applications.

This specific case is too narrow a focus, and its a perfect use case for extensions. The extension gets added and removed a second later, only one thing will ever check it.  Extensions even have list-only forms for this kind of purpose, making them also ultra cheap to add and check for

Don't add new behaviour to everything for the sake of one limited application